components/export/api/src/main/resources/ApplicationResources.properties

```diff
@@ -6,14 +6,89 @@ phenotips.exportPreferences.field.referrer=Referrer (report creator)
 phenotips.exportPreferences.field.creationDate=Report creation date
 phenotips.exportPreferences.field.author=Last modified by
 phenotips.exportPreferences.field.date=Last modification date
-phenotips.exportPreferences.field.genes=Candidate genes list
-phenotips.exportPreferences.field.genesComments=Candidate genes list comments
-phenotips.exportPreferences.field.rejectedGenes=Rejected genes.
-phenotips.exportPreferences.field.rejectGenesComments=Rejected genes comments.
+phenotips.exportPreferences.field.genes=Genes list
```

Comment:
I confirmed with Orion as a native speaker and apparently it should be "Gene list" not "Genes list"